SimCalorimetry/HcalSimProducers/python/hcalSimParameters_cfi.py

```diff
@@ -120,10 +124,14 @@
 run2_HE_2017.toModify( hcalSimParameters,
     he = dict(
         photoelectronsToAnalog = cms.vdouble([57.5]*14),
-        pixels = cms.int32(27370), 
-        sipmDarkCurrentuA = cms.double(0.055),
-        sipmCrossTalk = cms.double(0.32),
         doSiPMSmearing = cms.bool(True),
+        sipmTau = cms.double(10.),
+    )
+)
+from Configuration.Eras.Modifier_run2_HCAL_2017_cff import run2_HCAL_2017
+run2_HCAL_2017.toModify( hcalSimParameters,
+    ho = dict(
+        doSiPMSmearing = cms.bool(True)
```

Comment:
Well, if it's a bug or even an improvement to simulation to match data better, it better be fixed.
I would only prefer that all Run2/Run1 performance changes are factorized in a separate PR.
Run2 <2016 is still not affected by this PR.